source-control/git-guidance/readme.md

```diff
@@ -0,0 +1,168 @@
+# Git Guidance
+
+## The basics
+
+### Installation
+
+[Install Git](https://git-scm.com/downloads) and follow the [First-Time Git Setup](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup).
+
+### Cloning
+
+<!-- TODO: add more information about cloning repos (e.g., SSH, https, etc.) -->
```

Comment:
I prefer if we think of master as always shippable, i.e. not leaving empty titles with TODO:s but instead creating tasks to add these items separately. 

As per our own guideline above "Commit complete and well tested code. Never commit incomplete code"

Reply:
Done.